cirq-ionq/cirq_ionq/sampler_test.py

```diff
@@ -100,7 +100,7 @@ def test_sampler_multiple_jobs():
     results = sampler.sample(
         program=circuit,
         repetitions=4,
-        params=[cirq.ParamResolver({x: '0.5'}), cirq.ParamResolver({x: '0.6'})],
+        params=[cirq.ParamResolver({x: 0.5}), cirq.ParamResolver({x: 0.6})],
```

Comment:
This was coincidentally working before because the `value_of` code was calling `float` on the result after looking up in the param dict. But I think that behavior was unintentional because otherwise there would be no way to distinguish between strings used as symbols and strings containing floats. I can't think of any other place where we allow floats to be specified as strings, outside of serialization, and it certainly wasn't the intent to do so for parameter resolution.